arch/xtensa/include/esp32/esp32_himem_chardev.h

```diff
@@ -0,0 +1,42 @@
+/****************************************************************************
+ * arch/xtensa/include/esp32/esp32_himem_chardev.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Public Function Prototypes
+ ****************************************************************************/
+
+#ifndef __ESP32_HIMEM_CHARDEV_H
+#define __ESP32_HIMEM_CHARDEV_H
```

Comment:
Header guard not compliant to NuttX coding guidelines.
See the **Forming Guard Names** section at https://nuttx.apache.org/docs/latest/contributing/coding_style.html

Reply:
I agree.It's been revised.